make.js

```diff
@@ -454,6 +449,21 @@ function buildTask(taskName, taskListLength, nodeVersion) {
             rm('-Rf', taskTestsNodeModulesPath);
         }
     }
+
+    // remove duplicated task libs node modules from build tasks.
+    var buildTasksNodeModules = path.join(buildTasksPath, taskName, 'node_modules');
```

Comment:
Shouldn't we remove duplicate task-lib via the [make.json file](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/MavenV2/make.json#L5)?

Reply:
Removing duplicated task-lib via make.json file would require us to add it under Tasks\taskname\_buildConfigs files for each task and identify all tasks that are having duplicated task-libs. I prefer to remove duplicates via buildTask function.